src/main/java/com/minecolonies/coremod/achievements/package-info.java

```diff
@@ -3,8 +3,7 @@
  * <p>
  * Contains a subclass of:
  * <p>
- * - {@link net.minecraft.stats.Achievement}
- * - {@link net.minecraftforge.common.AchievementPage}
+//todo fill in as soon as existent
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)